benchmarks/data-caching/client/Dockerfile

```diff
@@ -8,12 +8,16 @@ ENV DEBIAN_FRONTEND noninteractive
 
 COPY memcached.tar.gz /memcached.tar.gz
 
-RUN apt-get update && apt-get install -y libevent-2.1-6 vim --no-install-recommends \
-	&& rm -rf /var/lib/apt/lists/*
-
-RUN buildDeps='curl gcc libc6-dev libevent-dev make' \
+RUN buildDeps='curl gcc libc6-dev libevent-dev make wget ca-certificates' \
 	&& set -x \
-	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends \
+	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends
+RUN wget https://github.com/libevent/libevent/releases/download/release-2.1.11-stable/libevent-2.1.11-stable.tar.gz \
```

Comment:
Why not install `libevent` using `apt-get`?

Reply:
libevent not present on risc-v

Reply:
Hmmm. 🤔 
Let's see how long this is going to take, if it takes too long to compile then we should do the same thing we did with ffmpeg.

Reply:
It should finish quickly as I have only built for amd64. 
Yes, it takes quite some time for risc-v, we can follow the same approach as with ffmpeg.

Reply:
I see.
Then let's do the same thing here too for now.
* have it installed using `apt-get` (for both ARM and x86), we leave out RISC-V for now
* you can create another PR that addresses this and fixes it for RISC-V, along with the one for `ffmpeg`